src/main/java/jp/kusumotolab/kgenprog/project/jdt/StatementListVisitor.java

```diff
@@ -57,6 +58,23 @@ public void analyzeStatement(CompilationUnit unit) {
   }
 
   private void consumeStatement(Statement s) {
+
+    if (s instanceof Block) {
+      final Block block = (Block) s;
+      final List statements = block.statements();
```

Comment:
IntelliJの警告が出るので、以下でお願いします。

```suggestion
      final List<?> statements = block.statements();
```